### ISSUE TITLE
install command on autoit3 version check problems

### ISSUE THREAD

**Issue by genius257** [genius257/au3pm#10]
```
au3pm.exe  install autoit ">=3.3.14.4"
```
expected: `3.3.16.0`
actual: `3.3.14.5`

```
au3pm.exe  install autoit
```
expected: `3.3.16.0`
actual: `3.3.14.5`

**Comment by genius257** [comment]
the dependency "autoit" will be deprecated and replaced with "autoit3".
This will avoid the 4 digits version number issue, and allow semver ranges to work as expected.
"autoit" will not ONLY allow the exact version string, so no ranges and all 4 version digits must be present.

**Comment by genius257** [comment]
fixed with 52f5b8d96d3704574ed713e855c0a165dc4d4261 and 20bc38d58ed25cc36af60fa2130f30ffbbbc40d7
